src/tokenizer/state.js

```diff
@@ -6,20 +6,15 @@ import { types as tt } from "./types";
 
 export default class State {
   init(options: Object, input: string) {
-    this.strict = options.strictMode === false ? false : options.sourceType === "module";
+    this.strict = options.strictMode === false
+      ? false
+      : options.sourceType === "module";
 
     this.input = input;
 
     this.potentialArrowAt = -1;
 
-    this.inMethod =
-      this.inFunction =
-      this.inGenerator =
-      this.inAsync =
-      this.inPropertyName =
-      this.inType =
-      this.noAnonFunctionType =
-        false;
+    this.inMethod = this.inFunction = this.inGenerator = this.inAsync = this.inPropertyName = this.inType = this.noAnonFunctionType = false;
```

Comment:
I had no idea people would chain this kind of calls! We could support it in prettier but that would add quite a bit of complexity. Since you are already writing them each on their own line, I think that it would be better to just rewrite is as

```js
this.inMethod = false;
this.inFunction = false;
// ...
```